protractor.conf.js

```diff
@@ -2,24 +2,32 @@ exports.config = {
 
     baseUrl: 'http://todomvc.com',
 
+    seleniumAddress: 'http://localhost:4444/wd/hub/',
```

Comment:
Is there any reason why we should favour starting the selenium server manually rather than leaving it to Protractor?

```
seleniumServerJar: require.resolve('protractor/node_modules/webdriver-manager/selenium/selenium-server-standalone-2.53.1.jar'),
``` 

I agree that in production environments we'd probably prefer to connect to an existing selenium grid, but I'm worried that a requirement for starting up selenium manually in a tutorial might introduce additional complexity.